presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryOutputMetadata.java

```diff
@@ -21,26 +21,26 @@
 
 public class QueryOutputMetadata
 {
-    private final String connectorId;
+    private final String catalogName;
     private final String schema;
     private final String table;
 
     private final Optional<String> connectorOutputMetadata;
     private final Optional<Boolean> jsonLengthLimitExceeded;
 
-    public QueryOutputMetadata(String connectorId, String schema, String table, Optional<String> connectorOutputMetadata, Optional<Boolean> jsonLengthLimitExceeded)
+    public QueryOutputMetadata(String catalogName, String schema, String table, Optional<String> connectorOutputMetadata, Optional<Boolean> jsonLengthLimitExceeded)
     {
-        this.connectorId = requireNonNull(connectorId, "connectorId is null");
+        this.catalogName = requireNonNull(catalogName, "catalogName is null");
         this.schema = requireNonNull(schema, "schema is null");
         this.table = requireNonNull(table, "table is null");
         this.connectorOutputMetadata = requireNonNull(connectorOutputMetadata, "connectorOutputMetadata is null");
         this.jsonLengthLimitExceeded = requireNonNull(jsonLengthLimitExceeded, "jsonLengthLimitExceeded is null");
     }
 
     @JsonProperty
-    public String getConnectorId()
+    public String getCatalogName()
```

Comment:
I'm pretty sure this is a backwards incompatible change.  Maybe we should leave the old method, but deprecated?  cc @electrum

Reply:
Two potential issues:

1. This will break the code for anyone who has an event listener and is using the getter.
2. This will change the JSON output for anyone who has an event listener and is serializing the object. We have code that does this at FB.

Reply:
cc @raghavsethi